weekly-mission(week2)/src/features/signUp.js

```diff
@@ -0,0 +1,21 @@
+import { addErrorMsgToMismatchConfirm } from "../view/addErrorMessageToInput.js";
+
+export const signUpHandler = (e) => {
+  e.preventDefault();
+  addErrorMsgToMismatchConfirm();
```

Comment:
사소하지만 붙어있으면 같은 맥락을 공유하는 듯한 느낌이 들어서 한칸 띄워주면 좋을 것 같아요~
```suggestion
  e.preventDefault();
  
  addErrorMsgToMismatchConfirm();
```

Reply:
개행을 언제 넣어야 할 지 헷갈렸는데, 맥락 기준으로 넣으면 되는 거였군요 !
근데 또 생각해보니 맥락이라는 것이 좀 범위가 헷갈리는 느낌도 들어서 질문 하나만 더하겠습니다 !
저는 이번에 코드 작성하면서 if문 앞과 뒤는 웬만하면 개행을 넣었던 편인 것 같은데요, 
if문이나 for문 같은 경우에도 앞 뒤 코드가 같은 맥락을 공유한다면 개행 없이 적는 것이 맞는거겠죠?

Reply:
맥락이라는게  `이 코드가 어떤 일을 하고 있는가!` 요런 느낌인데요!
if문과 for문 같은 경우에는 여기서는 약간 예외로 무조건 앞 뒤 개행을 넣어주는게 좋을 것 같아요!